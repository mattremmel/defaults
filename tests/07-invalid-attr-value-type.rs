use defaults::Defaults;

#[derive(Defaults)]
struct Foo {
    #[default(value = 10)]
    x: usize,
    y: usize,
}

fn main() {}
