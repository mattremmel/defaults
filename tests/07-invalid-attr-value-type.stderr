error: expected `default(value = "...")`
 --> $DIR/07-invalid-attr-value-type.rs:5:15
  |
5 |     #[default(value = 10)]
  |               ^^^^^^^^^^
